README.md

```diff
@@ -6,14 +6,14 @@ _If you like this project, please leave me a star._ &#9733;
 
 ## Algorithms
 
-|  #  |      Title     |   Solutions   | Video  | Difficulty  | Tag                   
-|-----|----------------|---------------|--------|-------------|-------------
-|1630|[Arithmetic Subarrays](https://leetcode.com/problems/arithmetic-subarrays/)|[Solution](../master/src/main/java/com/fishercoder/solutions/_1630.java) ||Sort|Medium|
-|1626|[Best Team With No Conflicts](https://leetcode.com/problems/best-team-with-no-conflicts/)|[Solution](../master/src/main/java/com/fishercoder/solutions/_1626.java) ||DP|Medium|
-|1625|[Lexicographically Smallest String After Applying Operations](https://leetcode.com/problems/lexicographically-smallest-string-after-applying-operations/)|[Solution](../master/src/main/java/com/fishercoder/solutions/_1625.java) ||BFS, DFS|Medium|
-|1624|[Largest Substring Between Two Equal Characters](https://leetcode.com/problems/largest-substring-between-two-equal-characters/)|[Solution](../master/src/main/java/com/fishercoder/solutions/_1624.java) ||String|Easy|
-|1620|[Coordinate With Maximum Network Quality](https://leetcode.com/problems/coordinate-with-maximum-network-quality/)|[Solution](../master/src/main/java/com/fishercoder/solutions/_1620.java) ||Greedy|Medium|
-|1619|[Mean of Array After Removing Some Elements](https://leetcode.com/problems/mean-of-array-after-removing-some-elements/)|[Solution](../master/src/main/java/com/fishercoder/solutions/_1620.java) ||Array|Easy|
+|  #  |      Title     |   Solutions   | Video  | Difficulty  | Tag                  
+|-----|----------------|---------------|--------|------|-------------
```

Comment:
Please undo this line of change as the current one lines up better with the headings.

Reply:
@fishercoder1534 I believe that is fixed!